### PR TITLE
Updated attendance

### PR DESCRIPTION
**Pull request by bahull**

### CODE REVIEW

**Comment by m2mathew** [comment]
@bahull Let's use single quotes in the app (except in JSX/HTML).

Address that and fix the conflicts, and I will merge! 😃

**Comment by bahull** [comment]
Updated to single quotes.

File: src/shared/december2017data.js [hunk 1]
```diff
@@ -19,27 +20,32 @@ const december2017data = [
...
     name: 'Dallas Stars',
     twitter: 'DallasStars',
-    about: 'Hat tricks a plenty',
+    about: 'Hat tricks a plenty'
   },
   {
     name: 'Dallas Cowboys',
     twitter: 'dallascowboys',
-    about: 'America\'s team for American-style football',
+    about: "America's team for American-style football"
```
Comment (m2mathew) [inline src/shared/december2017data.js]: Let's leave single quotes here, so we should have:
`'America\'s team for American-style football'`

**Comment by bahull** [comment]
Added in hanging commas. Definitely good to see how a formatter (prettier) can cause problems while trying to contribute!

**Comment by m2mathew** [comment]
👍 Sweet!

**Pull request merged**